.eslintrc.yaml

```diff
@@ -24,18 +24,16 @@ plugins:
   - 'prettier'
   - 'prefer-object-spread'
   - 'react'
-  - 'react-percy'
 overrides:
   - files: ['*.spec.js']
     rules:
       'react/display-name': 0
-  - files: ['*.percy.js', 'test/percy/suite.js']
-    env:
-      'react-percy/globals': true
+  - files: ['*.example.js']
```

Comment:
Maybe using `.visualexample.js` would help see the connection between `visualspec.js` and `visualexample.js` a bit more?